packages/dev/s2-docs/src/Layout.tsx

```diff
@@ -87,91 +87,98 @@ export function Layout(props: PageProps & {children: ReactElement<any>}) {
       </head>
       <body
         className={style({
-          display: 'flex',
-          flexDirection: 'column',
-          alignItems: 'center',
-          maxWidth: {
-            default: 'full',
-            lg: 1280
-          },
-          marginX: 'auto',
-          marginY: 0,
-          padding: {
-            default: 0,
-            lg: 12
-          },
-          paddingBottom: 0,
-          gap: {
-            default: 0,
-            lg: 12
-          },
+          margin: 0,
+          padding: 0,
           overscrollBehavior: {
             default: 'auto',
             lg: 'none'
           }
         })}>
-        <Header pages={pages} currentPage={currentPage} />
-        <MobileHeader
-          toc={<MobileToc key="toc" toc={currentPage.tableOfContents ?? []} />}
-          nav={<MobileNav key="nav" pages={pages} currentPage={currentPage} />} />
-        <div className={style({display: 'flex', width: 'full'})}>
-          <Nav pages={pages} currentPage={currentPage} />
-          <main 
-            key={currentPage.url}
-            style={{borderBottomLeftRadius: 0, borderBottomRightRadius: 0}}
-            className={style({
-              isolation: 'isolate',
-              backgroundColor: 'base',
-              padding: {
-                default: 12,
-                lg: 40
-              },
-              borderRadius: {
-                default: 'none',
-                lg: 'xl'
-              },
-              boxShadow: {
-                lg: 'emphasized'
-              },
-              width: 'full',
-              boxSizing: 'border-box',
-              flexGrow: 1,
-              display: 'flex',
-              justifyContent: 'space-between',
-              position: 'relative',
-              height: {
-                lg: '[calc(100vh - 72px)]'
-              },
-              overflow: {
-                lg: 'auto'
-              }
-            })}>
-            <article
+        <div
+          className={style({
+            isolation: 'isolate',
```

Comment:
new div to add isolation: isolate. This makes the sticky header appear behind dialogs.